NewForce-Cohort-6/tabloidfullstack-mashed-potatoes
# 4ViewCurrentUsersPosts

# Description
[Trello Ticket #4](https://trello.com/c/tuO0pf6r)

Added functionality to view a list of the logged-in user's approved posts, each with its title, author and category. These posts all have a publication date that is in the past and the list is sorted in order of publication date with the most recent on top.

# Testing Instructions
1. `git add` and `git commit` whatever you're working on on your branch
2. `git checkout main`
3. `git fetch --a`
4. `git checkout 3ViewAllPosts`
5. If you don't see the most recent version of my files, `git pull origin 3ViewAllPosts`
6. If you haven't created the Tabloid database, please do so.
Use this data to fill out tables and columns: [SQL Tables](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/01_Tabloid_Create_DB.sql)
Use this to seed the database : [SQL SEED DATA](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/02_Tabloid_Seed_Data.sql)
7. Click the execute with debugger solid green button to launch the app
8. If you are already logged in to the site, log out.
9. Log into the website using “[gteanby6@craigslist.orgx](mailto:gteanby6@craigslist.orgx)” (this user has 5 posts) 
10. From the home page, click the “My Posts” option in the navbar
11. The url should change to `/myposts` and a list of 5 posts should appear, each with a title in bold, an author, and a category
12. Log out of the site and log back in with “[foo@bar.com](mailto:foo@bar.com)” (this user has 0 posts)
10. From the home page, click the “My Posts” option in the navbar
11. The url should change to `/myposts` and the page should display "No posts yet" in the middle of the page



# Checklist:
